src/luger.hrl

```diff
@@ -7,9 +7,10 @@
 -define(INFO, 6).
 -define(DEBUG, 7).
 
--record(config, { app    :: string(),
-                  host   :: string(),
-                  statsd :: boolean() }).
+-record(config, { app       :: string(),
+                  host      :: string(),
+                  statsd    :: boolean(),
+                  max_msg_len :: integer() }).
```

Comment:
typespec should be updated to also include `undefined` as an allowed type.

Reply:
Yeah, I missed that one in my edits. Thanks for pointing it out.